README.md

```diff
@@ -26,7 +26,7 @@ To fetch a timestamp:
 1. Build client: `go build ./cmd/timestamp-cli`
 1. Create test blob to sign: `echo "myblob" > myblob`
 1. Fetch timestamp: `./timestamp-cli --timestamp_server http://localhost:3000 timestamp --hash sha256 --artifact myblob --out response.tsr`
-1. Verify timestamp: `openssl ts -verify -in response.tsr -data "myblob" -CAfile ts_chain.pem`
+1. Verify timestamp: `./timestamp-cli verify --timestamp response.tsr --data "myblob" --cert-chain ts_chain.pem`
 1. Inspect timestamp: `openssl ts -reply -in response.tsr -text`
```

Comment:
Should we add in an inspect command? We can add this in another PR too, let's just keep the existing issue open then

Reply:
Yeah, that sounds like a good idea. Let's add it in a follow up PR.